### PR TITLE
Wglc processing

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-ietf-core-problem-details.md [hunk 1]
```diff
@@ -153,16 +160,23 @@ non-empty<M> = (M) .and ({ + any => any })
...
 A number of problem detail entries, the Standard Problem Detail
 entries, are predefined (more predefined details can be registered,
-see {{new-spdk}}):
+see {{new-spdk}}).
+
+Note that, unlike {{RFC7807}}, Concise Problem Details data items have
+no explicit "problem type".
+Instead, the category (or, one could say, Gestalt) of the problem can
+be understood from the shape of the problem details offered, we talk of
+a "problem shape" for short.
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md]: ❤️

File: draft-ietf-core-problem-details.md [hunk 2]
```diff
@@ -371,11 +400,11 @@ reference:
 Initial entries in this sub-registry are as follows:
 
 | Key value | Name | CDDL Type | Brief description | Reference |
-| -1 | title | text | short, human-readable summary of the problem type | RFCXXXX |
-| -2 | detail | text | human-readable explanation specific to this occurrence of the problem | RFCXXXX |
-| -3 | instance | ~uri | URI reference identifying specific occurrence of the problem | RFCXXXX |
-| -4 | response-code | uint .size 1 | CoAP response code | RFCXXXX |
-{: #spdk title="Initial Entries in Standard Problem Detail Key registry"}
+| -1 | title | text | short, human-readable summary of the problem shape | RFC XXXX |
+| -2 | detail | text | human-readable explanation specific to this occurrence of the problem | RFC XXXX |
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md]: CDDL type should be text / tag38

File: draft-ietf-core-problem-details.md [hunk 3]
```diff
@@ -371,11 +400,11 @@ reference:
 Initial entries in this sub-registry are as follows:
 
 | Key value | Name | CDDL Type | Brief description | Reference |
-| -1 | title | text | short, human-readable summary of the problem type | RFCXXXX |
-| -2 | detail | text | human-readable explanation specific to this occurrence of the problem | RFCXXXX |
-| -3 | instance | ~uri | URI reference identifying specific occurrence of the problem | RFCXXXX |
-| -4 | response-code | uint .size 1 | CoAP response code | RFCXXXX |
-{: #spdk title="Initial Entries in Standard Problem Detail Key registry"}
+| -1 | title | text | short, human-readable summary of the problem shape | RFC XXXX |
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md]: Just a note of colour.  What the HTTPbis WG has in its [style guide](https://httpwg.org/admin/editors/style-guide#self-references) about references to self is pretty cute.

BTW, CoRE should have a similar document in the core-wg pages :-)

File: draft-ietf-core-problem-details.md [hunk 4]
```diff
@@ -629,26 +658,26 @@ tunnel-7807 = {
...
-Create a Custom Problem Detail entry with key 7807.
+Create a Custom Problem Details entry with key 7807.
 Move the values for "type" and "status", if present, to the equivalent
-keys 0 and 1 of the Custom Problem Detail entry.
+keys 0 and 1 of the Custom Problem Details entry.
 Move all remaining key/value pairs (additional members as per {{Section
 3.2 of RFC7807}}) in the converted {{RFC7807}} problem
 details object to the Custom Problem Details map unchanged.
 
 The inverse direction, carrying Concise Problem Details in a Problem
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md:720]: Maybe we could stick a ref to 8075 here, noting that there may be some sort of URI mapping involved for the `instance` field to make it usable cross-proxy.

File: draft-ietf-core-problem-details.md (hunk 2 above)
Comment (cabo) [reply draft-ietf-core-problem-details.md]: Sorry, I was too slow...

File: draft-ietf-core-problem-details.md [hunk 5]
```diff
@@ -403,7 +432,7 @@ Reference:
 Initial entries in this sub-registry are as follows:
 
 | Key value | Name          |  Brief description                                                     | Reference |
-|      7807 | tunnel-7807   |  Carry RFC 7807 problem details in a Concise Problem Details data item | RFCXXXX   |
+|      7807 | tunnel-7807   |  Carry RFC 7807 problem details in a Concise Problem Details data item | RFC XXXX   |
 {: #cpdk title="Initial Entries in Custom Problem Detail Key registry"}
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md]:
```suggestion
{: #cpdk title="Initial Entries in the Custom Problem Detail Key registry"}
```

Comment (cabo) [reply draft-ietf-core-problem-details.md]: Not an incorrect change, but makes we wonder: captions tend to be telegram style (not the app :-), so leaving out articles generally should be OK.

**Pull request merged**